shared/src/test/scala/mlscript/DiffTests.scala

```diff
@@ -318,9 +320,15 @@ class DiffTests extends org.scalatest.funsuite.AnyFunSuite {
             def showResult(prefixLength: Int) = {
               results match {
                 case R(S(head :: next)) =>
-                  head.zipWithIndex foreach { case (s, i) =>
-                    if (i == 0) output(" " * prefixLength + "= " + s)
-                    else output(" " * (prefixLength + 2) + s)
+                  val text = head match {
+                    case L(err) =>
+                      output("Unexpected runtime error")
```

Comment:
I'd change this to just `"Runtime error"` when the code block has already failed to type check. The error isn't exactly unexpected 😛